lib/rubocop/cop/rspec/empty_line_after_final_let.rb

```diff
@@ -28,13 +28,8 @@ def on_block(node)
           latest_let = node.body.child_nodes.select { |child| let?(child) }.last
 
           return if latest_let.nil?
```

Comment:
`return unless latest_let`?
This #955 has caused an avalanche. :D

Reply:
I’m not sure #955 is related to your suggestion.

Reply:
Right, I seem to have mixed things up. But there was something that triggered a chain reaction of refactorings.